Updating README.md

Arjit222830 opened the pull request:
Adding Contributor Shield Image to README.md file

Arjit222830 commented:
![WhatsApp Image 2021-03-11 at 12 39 48 PM (1)](https://user-images.githubusercontent.com/43830893/110749661-6dc80700-8267-11eb-8b74-8067816f2043.jpeg)

Arjit222830 commented:
![WhatsApp Image 2021-03-11 at 12 39 48 PM](https://user-images.githubusercontent.com/43830893/110749786-8fc18980-8267-11eb-9fd9-71eceacc27c4.jpeg)

ricknjacky reviewed (changes requested):
What exactly is this PR fixing? or enrich the pre-existing markdown files that you have changed here?

walterbender commented:
I like the idea of this, but I am a bit confused by all of the commits. Also, is there a way to not have the external dependency?

Arjit222830 commented:
@ricknjacky I have added a component which will show number of contributors in a markdown dynamically.

Arjit222830 commented:
@walterbender Sir, all these commit I did to ensure that the image look good on markdown files. That made me to do more commits than I should have done. I apologise for that. Its an external dependency which provide badges for github repository. If you like this idea, there are more badges like number of forks, issues, license etc. that I could contribute to this repository.

walterbender commented:
Maybe something for the license would be good. Not sure we need the rest as much of that is provided by github.

Arjit222830 commented:
@walterbender Sir, I have added 3 more badges: license as you suggested and in addition to that I also added fork and issues badges too. If you like them, plz add it
![music](https://user-images.githubusercontent.com/43830893/110886056-fbf6c880-830d-11eb-9b6c-1aba453a55eb.png)
.

ricknjacky commented:
> @ricknjacky I have added a component which will show number of contributors in a markdown dynamically.

My bad, I missed this. Looks great though! Thanks for the clarification.

Arjit222830 commented:
@walterbender  Sir, do u like this idea.. Plz respond

walterbender commented:
As I said previously, I think the licence and contributors badges are nice. I think the others are not necessary as they are easy to find on the github page.

Arjit222830 commented:
Made another pull request as I over committed a two changes in one Branch
